File: I-FIT/src/app/Modelo/Apprentice.ts
import { JourneyEnum } from "./Journey";
import { Routine } from "./Routine";

export class Apprentice{

    id: string;
    document: number;
    email: string;
    photo: string;
    password: string;
    name: string;
    journeyEnum: JourneyEnum;
    weight: number;
    age: number;
    height: number;
    comments: string;
    limitations: string;
    routine: Routine;
}

File: I-FIT-BACKEND/src/main/java/org/i_fit/entities/responsesgym/Apprentice.java
package org.i_fit.entities.responsesgym;

import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Data
@Document(collection = "apprentice")
public class Apprentice {

    @Id
    public Integer document;
    private String email;
    private String password;
    private String name;
    private Journey.JourneyEnum journeyEnum;
    private Integer weight;
    private Integer age;
    private Float height;
    private String comments;
    private String limitations;
    private Routine routine;





}
